File: src/WebUI/src/app/modules/capital/models/capital-response.ts
export interface CapitalResponse {
  id: number;
  name: string;
  balance: number;
  currency: string;
  includeInTotal: boolean;
  onlyForSavings: boolean;
  totalIncome: number;
  totalExpense: number;
  totalTransferIn: number;
  totalTransferOut: number;
}

File: src/Deed/Deed.Application/Capitals/Responses/CapitalResponse.cs
namespace Deed.Application.Capitals.Responses;

public sealed record CapitalResponse(
    int Id,
    string Name,
    decimal Balance,
    string Currency,
    bool IncludeInTotal,
    bool OnlyForSavings,
    decimal TotalIncome,
    decimal TotalExpense,
    decimal TotalTransferIn,
    decimal TotalTransferOut,
    DateTimeOffset CreatedAt,
    string? CreatedBy
);
